### PR TITLE
Feature/baddicom

### PR DESCRIPTION
**Pull request by tznind**
This feature removes all our bespoke wierd random tag generation code and seed files etc and instead uses our new package BadMedicine.Dicom.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -24,6 +24,8 @@ jobs:
     before_install:
     - docker run --name=mssql-server -e 'ACCEPT_EULA=Y' -e 'MSSQL_SA_PASSWORD=YourStrong!Passw0rd'
       -p 1433:1433 -d mcr.microsoft.com/mssql/server:2017-latest
+    - sudo apt install libc6-dev
```
Comment (rkm) [inline .travis.yml]: I don't think sudo is required by Travis as it runs as root by default

File: tests/microservices/Microservices.DicomRelationalMapper.Tests/DicomDatasetCollectionSourceTests.cs [hunk 2]
```diff
@@ -47,6 +55,8 @@ public void SourceReadSimpleTagToTable()
         public void TestStringTooLong(DataTooWideHandling strategy)
         {
             var ds = new DicomDataset();
+            
+            ds.AutoValidate = false;
```
Comment (rkm) [inline tests/microservices/Microservices.DicomRelationalMapper.Tests/DicomDatasetCollectionSourceTests.cs]: I assume this is needed to to BadMedicine using fo-dicom >= 4.0.2? This should be fine for the test packages, but we need to test the effect of the auto-validation before updating the main src.

**Comment by rkm** [comment]
This looks like it will work as-is, however I'm not sure about the separate solution files. Are they just there to avoid packing the test projects? I think there's a better way to do this if that's the only reason.

File: tests/microservices/Microservices.DicomRelationalMapper.Tests/DicomDatasetCollectionSourceTests.cs (hunk 2 above)
Comment (tznind) [reply tests/microservices/Microservices.DicomRelationalMapper.Tests/DicomDatasetCollectionSourceTests.cs]: Agreed.  We are also currently (with this PR) building and testing on different versions of fo-dicom which is not ideal.

**Pull request merged**